modules/stochastic_tools/doc/content/modules/stochastic_tools/install_pytorch.md

```diff
@@ -0,0 +1,164 @@
+# Install MOOSE with Pytorch C++ API (libtorch)
```

Comment:
I think a lot of this information is redundant. In general, we should send people to the general instructions and then explain the setup script. 

All of this detail is bound to be forgotten in the future if we decide to change the general install instructions and forget about this page.

I also think that this should go in the framework documentation and not stochastic_tools.

Reply:
Modified!